CMakeLists.txt

```diff
@@ -165,6 +169,13 @@ if(NOT NONET)
   endif()
 endif()
 
+if(DEVILUTIONX_SYSTEM_LIBFMT)
+  set(fmt_USE_STATIC_LIBS ${DEVILUTIONX_STATIC_LIBFMT})
+  find_package(fmt REQUIRED)

```

Comment:
For this to work, we need to provide a finder module at `CMake/Findfmt.cmake`
Example https://github.com/ceph/ceph/blob/master/cmake/modules/Findfmt.cmake

Reply:
That one is distributed under the LGPL license. I suppose that would be an issue? Looking for another one. A bit odd that `fmt` doesn't provide one.

Reply:
`fmt` does provide a cmake config but not all distributions install it

```
$ dpkg -L libfmt-dev | grep cmake
/usr/lib/x86_64-linux-gnu/cmake
/usr/lib/x86_64-linux-gnu/cmake/fmt
/usr/lib/x86_64-linux-gnu/cmake/fmt/fmt-config-version.cmake
/usr/lib/x86_64-linux-gnu/cmake/fmt/fmt-config.cmake
/usr/lib/x86_64-linux-gnu/cmake/fmt/fmt-targets-none.cmake
/usr/lib/x86_64-linux-gnu/cmake/fmt/fmt-targets.cmake
```

Both Debian/Ubuntu and buildroot fmt packages provide these files, so perhaps it's OK to not worry about Findfmt